src/doc/src/reference/manifest.md

```diff
@@ -341,7 +341,7 @@ incremental = true # whether or not incremental compilation is enabled
 overflow-checks = true # use overflow checks for integer arithmetic.
                    # Passes the `-C overflow-checks=...` flag to the compiler.
 
-# The release profile, used for `cargo build --release`.
+# The release profile, used for `cargo build --release` (and the dependencies for `cargo test --release`,  including the local lib/bin).
```

Comment:
One minor nit, "local lib/bin" might be confusing. Maybe just say "library or binary".